<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Alocacao</class>
 <widget class="QMainWindow" name="Alocacao">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>570</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Agendar festas</string>
  </property>
  <property name="windowIcon">
   <iconset resource="mures.qrc">
    <normaloff>:/imagem/imagem/aloca.jpg</normaloff>:/imagem/imagem/aloca.jpg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_agendar">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>260</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Agendar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Agendar Festa</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView_agenda">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>661</width>
      <height>191</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>280</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Festas agendadas</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>661</width>
      <height>185</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Data da festa</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Tema da festa</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Quantidade de pessoas</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Hora de inicio da festa</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Hora do fim da festa</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Desconto de entrada</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Valor de entrada</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLineEdit" name="lineEdit_dataFesta">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_temaFesta"/>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox_qtdPessoas">
           <property name="maximum">
            <number>800</number>
           </property>
           <property name="value">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTimeEdit" name="timeEdit_inicio"/>
         </item>
         <item>
          <widget class="QTimeEdit" name="timeEdit_fim"/>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="doubleSpinBox_desconto">
           <property name="decimals">
            <number>2</number>
           </property>
           <property name="maximum">
            <double>1000.000000000000000</double>
           </property>
           <property name="value">
            <double>0.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="doubleSpinBox_valorEntrada">
           <property name="maximum">
            <double>1000.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCalendarWidget" name="calendarWidget">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;duplo clik para confirmar data&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>710</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="mures.qrc"/>
 </resources>
 <connections/>
</ui>
